.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Library version and API version (please complete the following information):**
+ - Library: [e.g. 10.0.0]
+ - API [e.g. v6]
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
might also want to ask for relevant request / response logs (with a link to how to get log output) with PII removed

Reply:
👍🏻